client/components/data/query-whois/index.jsx

```diff
@@ -29,7 +29,10 @@ QueryWhois.propTypes = {
 	requestWhois: PropTypes.func.isRequired
 };
 
+const mapStateToProps = ( state, { domain } ) =>
+	( { requesting: isRequestingWhois( state, domain ) } );
```

Comment:
@mcfs: Is this an appropriate way to fix the function creation/binding problem in `connect()`?